plugins/telemetry-otel/src/test/java/org/opensearch/telemetry/OTelTelemetryPluginTests.java

```diff
@@ -42,7 +44,9 @@ public void setup() {
         // io.opentelemetry.sdk.OpenTelemetrySdk.close waits only for 10 seconds for shutdown to complete.
         Settings settings = Settings.builder().put(TRACER_EXPORTER_DELAY_SETTING.getKey(), "1s").build();
         oTelTracerModulePlugin = new OTelTelemetryPlugin(settings);
-        telemetry = oTelTracerModulePlugin.getTelemetry(null);
+        telemetry = oTelTracerModulePlugin.getTelemetry(
+            new TelemetrySettings(Settings.EMPTY, new ClusterSettings(settings, Set.of(TRACER_ENABLED_SETTING, TRACER_SAMPLER_PROBABILITY)))
```

Comment:
`Set.of` is good, the outcome of the test is no different then before:

```
telemetry = oTelTracerModulePlugin.getTelemetry(null);
```

So either revert the change or add assertions in `testGetTelemetry`

Reply:
Updating it to `telemetry = oTelTracerModulePlugin.getTelemetry(null);` will fail the UT as in ProbabilisticSampler we have null check for telemetrySetting.

Reply:
> Please do correct me if I am missing something.

My apologies, should have checked that initialization (`setup`) fails without this settings being set, all good now